src/main/java/net/aufdemrand/denizen/events/entity/EntityKilledScriptEvent.java

```diff
@@ -172,12 +178,6 @@ public void onEntityKilled(EntityDamageEvent event) {
         entity = new dEntity(event.getEntity());
         damage = new Element(event.getDamage());
         final_damage = new Element(event.getFinalDamage());
-        // Check for possibility of death first
-        if (entity.isValid() && entity.isLivingEntity()) {
-            if (final_damage.asDouble() < entity.getLivingEntity().getHealth()) {
-                return;
-            }
-        }

```

Comment:
I actually.. haven't edited this at all.. Must not have merged from your repo.. O.o